packages/gluestick/src/commands/test/__tests__/test.test.js

```diff
@@ -0,0 +1,156 @@
+      context.logger.info.mockClear();
+      path.join = () => 'cwd/empty/package.json';
+      testCommand(getMockedContext({}, [
+        { test: /\.js$/ },
+        { test: /\.scss$/ },
+        { test: /\.css$/ },
+        { test: /\.woff$/ },
+        { test: /\.png$/ },
+        { test: /\.ttf$/ },
+      ]), {
+        debugTest: true,
+        parent: {
+          rawArgs: ['', '', '', '--debug-test', '-i'],
+        },
+      });
+      path.join = originalPathJoin;
+      expect(context.logger.info.mock.calls[0]).toEqual([
+        'Option \'-i\' is always set by default in debug mode',
+      ]);
+    });
+  });
+
+  describe('in non-debug mode', () => {
+    it('should run jest directly with default config', () => {
+      path.join = () => 'cwd/empty/package.json';
+      testCommand(getMockedContext({
+        alias1: 'path/to/alias1',
+        alias2: 'path/to/alias2',
+      }, [
+        { test: /\.js$/ },
+        { test: /\.scss$/ },
+        { test: /\.css$/ },
+        { test: /\.woff$/ },
+        { test: /\.png$/ },
+        { test: /\.ttf$/ },
+      ]), {
+        debugTest: false,
+        parent: {
+          rawArgs: ['', '', ''],
+        },
+      });
+      path.join = originalPathJoin;
+      expect(jestMock.run.mock.calls.length).toBe(1);
+      const jestConfig = JSON.parse(jestMock.run.mock.calls[0][0][1]);
+      expect(jestMock.run.mock.calls[0][0].indexOf('--config')).toBeGreaterThan(-1);
+      Object.keys(jestConfig.moduleNameMapper).forEach((mapper) => {
+        expect(
+          ['woff', 'css', 'alias1', 'alias2'].find((alias) => mapper.includes(alias)),
+        ).not.toBeUndefined();
+      });
+      expect(jestConfig.roots).toEqual(['src']);
+    });
+
+    it('should run jest directly with merged default and custom config', () => {
+      path.join = () => 'cwd/custom/package.json';
+      const originalExistsSync = fs.existsSync.bind(fs);
+      fs.existsSync = () => true;
+      testCommand(getMockedContext({
+        alias1: 'path/to/alias1',
+        alias2: 'path/to/alias2',
+      }, [
+        { test: /\.js$/ },
+        { test: /\.scss$/ },
+        { test: /\.css$/ },
+        { test: /\.woff$/ },
+        { test: /\.png$/ },
+        { test: /\.ttf$/ },
+      ]), {
+        debugTest: false,
+        parent: {
+          rawArgs: ['', '', ''],
+        },
+      });
+      path.join = originalPathJoin;
+      expect(jestMock.run.mock.calls.length).toBe(1);
+      const jestConfig = JSON.parse(jestMock.run.mock.calls[0][0][1]);
+      expect(jestMock.run.mock.calls[0][0].indexOf('--config')).toBeGreaterThan(-1);
+      Object.keys(jestConfig.moduleNameMapper).forEach((mapper) => {
+        expect(
+          ['woff', 'css', 'alias1', 'alias2'].find((alias) => mapper.includes(alias)),
+        ).not.toBeUndefined();
+      });
+      expect(jestConfig.roots).toEqual(['src', 'test', 'customRoot']);
+      expect(jestConfig.verbose).toBeTruthy();
+      fs.existsSync = originalExistsSync;
```

Comment:
I'd delete this (and above things related). Jest tests are sandboxed so you don't need here.